### PR TITLE
[DDW-1086] Fix phrasing of insufficient funds for tokens message

### PR DESCRIPTION
**Pull request by lucas-barros**
This PR fixes the phrasing of of insufficient funds for tokens message. 
Previous error message was `“Insufficient funds to support tokens. A minimum of {someAdaAmount} ADA must remain in the wallet after this transaction.”`.
It was incorrect because `{someAdaAmount}` is not the value that should remain after the transaction but the value missing so the transaction will leave enough ada in the wallet to support the tokens.

It also adds missings translations pointed out in this [PR](https://github.com/input-output-hk/daedalus/pull/2964).

## Todos

- [ ] TODO

## Screenshots
Screenshot from 2022-05-13 13-01-25
Screenshot from 2022-05-13 13-01-49

## Testing Checklist

- [Slack QA thread](https://input-output-rnd.slack.com/archives/GGKFXSKC6/p1650633987150719)
- [ ] Test

---

### Basics

- [x] PR assigned to the PR author(s)
- [x] `input-output-hk/daedalus-dev` and `input-output-hk/daedalus-qa` assigned as PR reviewers
- [ ] ~~If there are UI changes, Alexander Rukin assigned as an additional reviewer~~
- [x] All visual regression testing has been reviewed (assign `run Chromatic` label to PR to trigger the run)
- [x] PR has appropriate labels (`release-vNext`, `feature`/`bug`/`chore`, `WIP`)
- [x] PR link is added to a Jira ticket, ticket moved to In Review
- [x] PR is updated to the most recent version of the target branch (and there are no conflicts)
- [x] PR has a good description that summarizes all changes
- [x] PR contains screenshots (in case of UI changes)
- [x] CHANGELOG entry has been added to the top of the appropriate section (_Features_, _Fixes_, _Chores_) and is linked to the correct PR on GitHub
- [x] There are no missing translations (running `yarn manage:translations` produces no changes)
- [x] Text changes are proofread and approved (Jane Wild / Amy Reeve)
- [x] Japanese text changes are proofread and approved (Junko Oda)
- [x] Storybook works and no stories are broken (`yarn storybook`)
- [x] In case of dependency changes `yarn.lock` file is updated

### Code Quality

- [ ] Important parts of the code are properly commented and documented
- [ ] Code is properly typed with typescript types
- [ ] React components are split-up enough to avoid unnecessary re-renderings
- [ ] Any code that only works in main process is neatly separated from components

### Testing

- [ ] New feature/change is covered by acceptance tests
- [ ] New feature/change is manually tested and approved by QA team
- [ ] All existing acceptance tests are still up-to-date
- [ ] New feature/change is covered by Daedalus Testing scenario
- [ ] All existing Daedalus Testing scenarios are still up-to-date

### After Review

- [ ] Update Slack QA thread by marking it with a green checkmark

### CODE REVIEW

**Comment by miorsufianiohk** [comment]
Hi @daniloprates , could we confirm how do you want to proceed with thia PR as [discussed here](https://input-output-rnd.slack.com/archives/GGKFXSKC6/p1650633987150719) please, thank you 🙏

File: source/renderer/app/api/utils/apiHelpers.ts [hunk 1]
```diff
@@ -27,22 +27,23 @@ export const doesWalletRequireAdaToRemainToSupportTokens = (
   hasAssetsRemainingAfterTransaction?: boolean
 ): {
   requiresAdaToRemainToSupportNativeTokens: boolean;
-  adaToRemain?: number;
+  adaToProceed?: number;
 } => {
   const adaToProceedRegex = new RegExp(
-    /.*I need approximately([\s\d.,]+)ada to proceed.*/
+    /I need approximately\s+([\d.,]+)\s+ada to proceed/
```
Comment (szymonmaslowski) [inline source/renderer/app/api/utils/apiHelpers.ts:33]: Two small things:
1. The `new RegExp` construction is redundant here as you are passing a regex literal which itself is sufficient.
2. I am wondering if the number formating setting, which we can set in the profile settings applies to the number in that error message.. I guess that setting is only for our ui part and probably it is not sent to the cardano-wallet. Nevertheless, I think it is worth ensuring.

File: source/renderer/app/api/utils/apiHelpers.ts [hunk 2]
```diff
@@ -27,22 +27,23 @@ export const doesWalletRequireAdaToRemainToSupportTokens = (
...
   );
 
+  const [, adaToProceed] = adaToProceedRegex.exec(error.message) ?? [];
```
Comment (szymonmaslowski) [inline source/renderer/app/api/utils/apiHelpers.ts:36]: The `adaToProceed` variable shadows the function argument. Actually, what is the intention here? Should the `adaToProceed` from the error message take priority over the passed `adaToProceed`?

**Review by szymonmaslowski — CHANGES_REQUESTED** [review]

**Review by ManusMcCole — APPROVED** [review]

File: source/renderer/app/api/utils/apiHelpers.ts (hunk 2 above)
Comment (lucas-barros) [reply source/renderer/app/api/utils/apiHelpers.ts:34]: There is no adaToProceed argument. Could it be you are talking about the return type of the function?

Comment (szymonmaslowski) [reply source/renderer/app/api/utils/apiHelpers.ts:34]: Ahh, sorry. You are right. It was my misunderstanding. Everything is good ;)

**Review by szymonmaslowski — APPROVED** [review]

**Review by DominikGuzei — APPROVED** [review]
Code LGTM! 👏

**Pull request merged**